packages/features/eventtypes/components/EventTypeDescription.tsx

```diff
@@ -40,6 +42,31 @@ export const EventTypeDescription = ({
 
   const paymentAppData = getPaymentAppData(eventType);
 
+  const showManagedBadge = eventType.metadata?.managedEventConfig && !isPublic;
+  const numOfManagedByAdmins = eventType.managedBy?.admins.length || 0;
+  const showManagedByToolTip = showManagedBadge && numOfManagedByAdmins > 0;
+  let managedByToolTipContent = "";
+  let managedByToolTipContentWithLink;
```

Comment:
pls remove `managedByToolTipContentWithLink` and try to use `managedByToolTipContent` in both condition. Use single variable to store the tooltip content.

Reply:
ya done